rero_ils/config.py

```diff
@@ -1340,29 +1340,33 @@ def _(x):
     'documents', RERO_ILS_DEFAULT_AGGREGATION_SIZE)
 RECORDS_REST_FACETS = dict(
     documents=dict(
+        i18n_aggs=dict(
+            contribution=dict(
```

Comment:
Is it really working. This sound like nested aggregations to me.

Reply:
yes it works using `i18n_facets_factory()`.
If the key `i18n_aggs` exists, I retrieve only the aggregation according the language needed.